index.html

```diff
@@ -54,6 +57,28 @@ <h1 class="popheading">Tonic</h1>
                       </div>
             </div>    
            </div>
+           <div class="containerdesktop">
+            <div class="imgdivdesktop">
```

Comment:
- [ ] As stated by the previous reviewer, kindly 😃try to stick to the Figma Design I see that you want to put some margin left here to align the container here like the Figma 😊

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/215073243-3c9f4a62-9624-4a6c-ab01-5135c4afaf1b.png) |    ![image](https://user-images.githubusercontent.com/83360636/215072857-a854d037-ebaf-441f-907e-0146a2913e30.png) |

Reply:
Suggested changes made screen shot attached below for your kind reference.
![image](https://user-images.githubusercontent.com/117091923/215096358-b8d8b3c7-4065-416b-9d1a-c764a18b7ecf.png)